docker/local-node/entrypoint.sh

```diff
@@ -146,8 +146,22 @@ else
     zk contract initialize-validator
   fi
 
-  ## init hyperchain
-  zk contract register-hyperchain
```

Comment:
Is there a chance something else is relying on this? The new CLI perhaps?

Reply:
it is still executed - in line 155.

'the -z' option means that a given env variable is not set.